ACPs/125-basefee-reduction/README.md

```diff
@@ -0,0 +1,42 @@
+```text
+ACP: 125
+Title: Reduce C-Chain base fee to 1 Gwei.
+Author(s): Stephen Buttolph <https://github.com/StephenButtolph>, Darioush Jalali <https://github.com/darioush>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+This ACP proposes reducing the base fee on the Avalanche C-Chain from 25 Gwei to 1 Gwei.
+
+## Motivation
+
+With dynamic fees, the gas price is supposed to be a result of a continuous auction such that the consumed gas per second converges to the target gas usage per second.
+When dynamic fees were first introduced safeguards were introduced to ensure that the mechanism worked as intended, such as a relatively high minimum gas price and a maximum gas price.
+The maximum gas price has since been entirely removed. The minimum gas price has been reduced significantly. However, it is still too high, and is therefore reducing usage of the network.
```

Comment:
There should be an empty line between these

Reply:
done

Reply:
Instead of asserting that "it is still too high", I'd recommend we say that the fact that the observed base fee is often pinned to this minimum shows that it is higher than what the market demands, and therefore artificially reducing network usage.